src/cowboy_req.erl

```diff
@@ -726,6 +751,30 @@ set_resp_header(Name, Value, Req=#{resp_headers := RespHeaders}) ->
 set_resp_header(Name,Value, Req) ->
 	Req#{resp_headers => #{Name => Value}}.
 
+% @todo make it work with iodata(), for now only bitstrings are accepted
+
+-spec set_resp_headers_list(list({ binary(), iodata() }), Req)
+	-> Req when Req::req().
+set_resp_headers_list(HeaderTupleList, Req) ->
+	set_resp_headers_list(HeaderTupleList, #{}, Req).
+
+set_resp_headers_list([], Map, Req) ->
+	set_resp_headers(Map, Req);
+
+set_resp_headers_list([{<<"set-cookie">>, _} | Headers], Map, Req) ->
```

Comment:
@essen I wonder if to have some consistency with the `set_resp_header` we should instead throw an error.